lib/temporal/workflow/context.rb

```diff
@@ -287,6 +287,13 @@ def on_signal(&block)
         end
       end
 
+      def on_query(query = nil, &block)
+        query ||= Dispatcher::WILDCARD
```

Comment:
If you look at the signal implementation, it defaulted to `'signaled'` as the event_name when registering an unnamed handler. Why use the WILDCARD here?